nginx-ingress-istio/ingress-ingress-ns.yaml

```diff
@@ -0,0 +1,14 @@
+# ingress-ingress-ns.yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: nginx-ingress
+  namespace: ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/service-upstream: "true"
+    nginx.ingress.kubernetes.io/upstream-vhost: httpbin.default.svc.cluster.local
+spec:
+  backend:
+    serviceName: httpbin
+    servicePort: 8000
```

Comment:
@ZackButcher Also, currently there is only one httpbin service in the default namespace. The same httpbin service is exposed to external traffics by the two Ingress resources in the default and the ingress namespaces. Would it be better if we have two httpbin services in the default namespace, each exposed by a different Ingress resource?